### ISSUE TITLE
Find highlighting hard to see

### ISSUE THREAD

**Issue by derekperkins** [MordFustang21/zed-one-dark-pro#11]
In Goland

<img width="349" alt="image" src="https://github.com/MordFustang21/zed-one-dark-pro/assets/3588778/5ec29364-5e8a-4848-875c-3231bcde1e8f">

In Zed

<img width="396" alt="image" src="https://github.com/MordFustang21/zed-one-dark-pro/assets/3588778/4a418d38-69b5-4ff5-93bc-e00be588d967">

**Comment by 0x2CA** [comment]
```
    "experimental.theme_overrides": {
        "search.match_background": "#74ade866",
        "players": [
            {
                "cursor": "#74ade8ff",
                "background": "#74ade8ff",
                "selection": "#74ade83d"
            },
            {
                "cursor": "#be5046ff",
                "background": "#be5046ff",
                "selection": "#be50463d"
            },
            {
                "cursor": "#bf956aff",
                "background": "#bf956aff",
                "selection": "#bf956a3d"
            },
            {
                "cursor": "#b477cfff",
                "background": "#b477cfff",
                "selection": "#b477cf3d"
            },
            {
                "cursor": "#6eb4bfff",
                "background": "#6eb4bfff",
                "selection": "#6eb4bf3d"
            },
            {
                "cursor": "#d07277ff",
                "background": "#d07277ff",
                "selection": "#d072773d"
            },
            {
                "cursor": "#dec184ff",
                "background": "#dec184ff",
                "selection": "#dec1843d"
            },
            {
                "cursor": "#a1c181ff",
                "background": "#a1c181ff",
                "selection": "#a1c1813d"
            }
        ]
    }
```
![image](https://github.com/MordFustang21/zed-one-dark-pro/assets/19868292/0239b8e3-6e68-4d6a-9059-8cfda76d0f1f)

**Comment by MordFustang21** [comment]
This is fixed in v0.0.7 once this merges https://github.com/zed-industries/extensions/pull/1441
